Formula/zbar.rb

```diff
@@ -21,12 +39,13 @@ class Zbar < Formula
   depends_on "freetype"
   depends_on "libtool" => :run
 
-  # Fix JPEG handling using patch from
-  # https://sourceforge.net/p/zbar/discussion/664596/thread/58b8d79b#8f67
-  # already applied upstream but not present in the 0.10 release
-  patch :DATA
-
   def install
+    if build.head?
+      inreplace "configure.ac", "-Werror", ""
+      gettext = Formula["gettext"]
+      system "autoreconf", "-fvi", "-I", "#{gettext.opt_share}/aclocal"
```

Comment:
[gettext provides the `AM_ICONV` macro](https://www.gnu.org/software/gettext/manual/html_node/AM_005fICONV.html), which is used in the configure.ac.